lib/msal-common/src/utils/Constants.ts

```diff
@@ -11,6 +11,7 @@ export const Constants = {
     // default authority
     DEFAULT_AUTHORITY: "https://login.microsoftonline.com/common/",
     DEFAULT_AUTHORITY_HOST: "login.microsoftonline.com",
+    DEFAULT_AUTHORITY_TENANT: "common",
```

Comment:
Can we add "common" somewhere in this variable name?

Reply:
Added